### PR TITLE
Accessibility Dev Tools Setup

### PR DESCRIPTION
**Pull request by cameron-eyds**
*Issue #:* /bcgov/entity#17808

*Description of changes:*
- Implements axe-core, vue-axe and jest-axe
-axe-core, vue-axe:  Provides developer feedback at runtime for accessibility issues. 
Screenshot 2023-09-22 at 9 19 10 AM

- jest-axe: See the update to in the setup file and example test implementation 
- Example of a AA failure:
Screenshot 2023-09-22 at 9 22 27 AM

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the PPR license (Apache 2.0).

### CODE REVIEW

**Review by doug-lovett — APPROVED** [review]
Fantastic, thanks for the overview.

File: ppr-ui/src/main.ts [hunk 1]
```diff
@@ -58,6 +59,15 @@ async function start () {
     await initLdClient()
   }
 
+  // For now, ONLY run for local development
+  // In your local .env add: VUE_APP_LOCAL_DEV="true"
```
Comment (dimak1) [inline ppr-ui/src/main.ts:63]: 👍🏻

**Review by dimak1 — APPROVED** [review]
Awesome!

File: ppr-ui/src/components/common/QsAccessBtn.vue [hunk 2]
```diff
@@ -1,5 +1,5 @@
 <template>
-  <div id="qs-access-btn">
+  <div id="qs-access-btn" role="button" aria-label="qualified-supplier-access-request-btn">
```
Comment (cameron-eyds) [inline ppr-ui/src/components/common/QsAccessBtn.vue:2]: An example of an AA issue, picked up and subsequently fixed by our jest. Very minimal stuff and this is the vast majority of the type of issues.

File: ppr-ui/src/main.ts [hunk 3]
```diff
@@ -58,6 +59,15 @@ async function start () {
...
+  if (process.env.VUE_APP_LOCAL_DEV === 'true') {
```
Comment (cameron-eyds) [inline ppr-ui/src/main.ts:64]: As in the comment, if you want to run this locally, just add the env variable to your local env.

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://bcregistry-assets-dev--pr-1546-qzr6bkbd.web.app

**Pull request merged**